runtime/vm/resolvesupport.cpp

```diff
@@ -931,6 +931,7 @@ resolveInstanceFieldRefInto(J9VMThread *vmStruct, J9Method *method, J9ConstantPo
 		J9Class *flattenableClass = NULL;
 		J9FlattenedClassCache *flattenedClassCache = NULL;
 #endif		
+		bool isWithField = false;
```

Comment:
C++ file, so you can declare the variable when it is assigned below.

Reply:
If I move the declaration to where it is assigned below, I get a compilation error. There is a `goto illegalAccess` before the assignment.

Reply:
OK.